pkg/controller/persistentvolume/types.go

```diff
@@ -25,6 +25,17 @@ import (
 	"k8s.io/kubernetes/pkg/client/cache"
 )
 
+const (
+	// A PVClaim containing this annotation requests a quality of service tier.
+	// The value of this field is arbitrary and known by the user and administrator.
+	// This is an experimental feature and likely to change in the future.
+	qualityOfServiceRequestKey = "experimental/k8s.io/persistentVolumeProvisioning-qos"
```

Comment:
Sure thing.  I will re-introduce this with the provisioner.  It will look for this on a PVC and provision a volume for it and add the 'createdForKey' annotation.